examples/system/index.ts

```diff
@@ -0,0 +1,165 @@
+import './stats.d';
```

Comment:
Oh yes, it shouldn't be a "npm project" it should be a private project, as you can mark in your package. But at least it would be standalone and full-fletched, making reproduction easier I think. Now the demo is weirdly mixed into the code imo, making it harder to guess what parts are relevant for usage. Of course the way I had it before was even worse, but I prefer doing it in the way I linked things.